src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -365,16 +366,22 @@ class AdditionalDetailsStep extends React.Component {
                                 keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
                             />
                         </View>
-                        <FormAlertWithSubmitButton
-                            isAlertVisible={isErrorVisible}
-                            onSubmit={this.activateWallet}
-                            onFixTheErrorsLinkPressed={() => {
-                                this.form.scrollTo({y: 0, animated: true});
-                            }}
-                            message={errorMessage}
-                            isLoading={this.props.walletAdditionalDetails.isLoading}
-                            buttonText={this.props.translate('common.saveAndContinue')}
-                        />
+                        <SafeAreaConsumer>
+                            {({safeAreaPaddingBottomStyle}) => (
+                                <View style={[styles.flex1, safeAreaPaddingBottomStyle]}>
```

Comment:
Do we need this additional View? Can we put the styles directly on the Form Alert? Wanted to check since none of the other updates you made need a new component like this one.

Reply:
Yeah I just chucked this `View` on there so we could apply the padding. It looks like `FormAlertWithSubmitButton` has a `containerStyles` prop that could be used here.

To tell you the truth, while this is a nice catch - I'm kinda leaning towards :do-nothing:. 

This page is going to be refactored to use `Form` eventually and the form component will apply the padding by default when that time comes.